### ISSUE TITLE
curl returning `Not Found` for  `shopping_list_complete.rb`

### ISSUE THREAD

**Issue by brandonjyuhas** [stbarrientos/sample-teach#1]
When I open the files in Nano, there is only the text `Not Found`.

Similarly, if I enter the URL located in the instructions (https://raw.githubusercontent.com/stbarrientos/sample-teach/master/shopping_list_complete.rb), I get the text `Not Found`.

**Comment by stbarrientos** [comment]
Yes I renamed one of my files on the fly, luckily no one encountered that problem during the sample teach. I will update the readme when I can

**Comment by brandonjyuhas** [comment]
@stbarrientos did we ever get this fixed? It's blocking our release to production

**Comment by stbarrientos** [comment]
@brandonjyuhas This is still an open issue, I’ll have the team take a look
